src/main/java/seedu/address/logic/comparator/ApplicationDateAndInterviewDateComparator.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.comparator;
+
+import java.time.LocalDate;
+import java.util.Comparator;
+
+import seedu.address.model.internship.InternshipApplication;
+
+/**
+ * Comparator for sorting InternshipApplication by application date and interview date.
+ */
+public class ApplicationDateAndInterviewDateComparator implements Comparator<InternshipApplication> {
+    private final LocalDate todayDate;
+
+    public ApplicationDateAndInterviewDateComparator() {
+        todayDate = LocalDate.now();
+    }
```

Comment:
I think this is not needed as it is not used?

Reply:
I changed the 'ApplicationDate**Then**InterviewDateComparator' to 'ApplicationDate**And**InterviewDateComparator'(because i changed the implementation hence the name change) so im actually using the one with 'and' now, but I did forget to delete the other one. Thanks!!